mupen64plus-core/upstream/src/device/rcp/vi/vi_controller.c

```diff
@@ -170,6 +175,27 @@ void write_vi_regs(void* opaque, uint32_t address, uint32_t value, uint32_t mask
     masked_write(&vi->regs[reg], value, mask);
 }
 
+
+void resolution_reset_check(){
+    if(l_resolutionReset != 0){
+        // Making sure to pause once the core has started up;
+        // sometimes the graphics plugin finishes first
+        if(g_EmulatorRunning && !g_rom_pause) {
+            if(strncmp(get_gfx_name(),"GLideN64",7) == 0 && l_resolutionResetCoreCounter < 8) {
```

Comment:
In here you are adding to the core explicit knowledge of what the currently loaded video plugin is. This breaks the part of the core design which says that the core should have no knowledge of the graphics plugin. I would try to find another way of doing this.

Reply:
This was a temporary fix I seem to have forgotten to check which came from commit 'afa79e3c5b4f22ef9d8e939bf2e1290f9bec88b9', I'm testing without the get_gfx_name() function existing at all anymore and it still seems to work as intended, I'll continue to test when I settle some other things